src/VAC/VectorAnimationComplex/KeyEdge.h

```diff
@@ -97,9 +97,9 @@ class Q_VPAINT_EXPORT KeyEdge: public KeyCell, public EdgeCell
     friend class VAC;
 
     ~KeyEdge();
-    KeyVertex * startVertex_;
-    KeyVertex * endVertex_;
-    EdgeGeometry * geometry_;
+    KeyVertex * startVertex_ = nullptr;
+    KeyVertex * endVertex_ = nullptr;
+    EdgeGeometry * geometry_ = nullptr;
```

Comment:
yay! :)

Reply:
I spent a lot of time for detetecting the "wild pointers"...